### PR TITLE
Update to symfony messenger 4.3 (fix deprecations)

### PR DESCRIPTION
**Pull request by tacman**
Fixed composer.json and the reference to message_bus.

### CODE REVIEW

File: src/Domain/composer.json [hunk 1]
```diff
@@ -31,7 +31,7 @@
     "require-dev": {
         "doctrine/orm": "^2.6",
         "ramsey/uuid": "^3.7",
-        "symfony/messenger": "^4.2",
+        "symfony/messenger": "^4.3",
```
Comment (ro0NL) [inline src/Domain/composer.json]: can you also update the `conflict` rule above

Comment (tacman) [reply src/Domain/composer.json]: I've bumped it, and I think it got added correctly to the PR.  Alas, I'm not sure what the conflict rule does, so I just changed it from 4.2 to 4.3.  

Travis is still complaining, though it's on Code Quality, and it seems to be related to self and DoctrineCollection.

Comment (ro0NL) [reply src/Domain/composer.json]: see https://getcomposer.org/doc/04-schema.md#conflict

so we ensure you cant use msgphp with messenger lower than 4.3, in your app. Our require-dev section does not apply when installing msgphp as a dependency in your app, hence the conflict rule

**Comment by tacman** [comment]
I purged my vendor/msgphp directory, then did composer update (and composer
install), but couldn't clear the cache.  Not sure

In DefinitionErrorExceptionPass.php line 54:
!!

!!    Cannot autowire service "App\Console\ClassContextElementFactory":
argument
!!    "$passwordHashing" of method "__construct()" has type
"MsgPhp\User\Password
!!    \PasswordHashing" but this class was not found.

**Comment by ro0NL** [comment]
Oh ha, i merged a BC break in between :) https://github.com/msgphp/msgphp/pull/350

you can now user symfony's password encoder directly, see the new skeleton how to do it: https://github.com/msgphp/msgphp/pull/350/files#diff-58284f8a311ee2589f06fc47976551bd

**Comment by tacman** [comment]
Yep, works now!

**Pull request merged**